ui/packages/ui/src/app/components/connectorStepHelpers/FilterTreeComponent.tsx

```diff
@@ -10,8 +10,8 @@ import {
 import { CubesIcon, ExclamationCircleIcon } from "@patternfly/react-icons";
 import React from "react";
 import { PageLoader } from "..";
-import { ApiError } from "../../shared";
-import { WithLoader } from "../../shared/WithLoader";
+import { ApiError } from "shared";
+import { WithLoader } from "shared/WithLoader";
```

Comment:
I think ApiError and WithLoader can be combined into one import, if WithLoader is added to index.ts in shared.  There are probably other similar cases in other components where imports could be combined with addition to index.ts